### PR TITLE
Shade 2.1 :: add Meta knob & fx link inverse button

### PR DESCRIPTION
**Pull request by ronso0**
- [x] added Meta knob
- [x] added Meta/Super link inverse button
- [x] decreased the minimum width to 1008px
- [x] added/updated some tooltips (only did a quick check)

There's still an error showing up regarding a wrong size declaration in skin.xml. When I tried to fix that with the exact same values/policies, some gaps below decks changed...mysterious. I'll leave it as is for now, don't have the time to fiddle with that absolute positioned layout, it is quite a pain IMO

### CODE REVIEW

**Comment by ronso0** [comment]
Re: FX routing buttons
Initially I understood the aim is to have routing that assign FX unit 1 and/or 2 to a deck/bus side, right?
But this won't work in Shade because of the bus concept. It's possible to have a toggle that swaps the controls of FX1 or FX2, but assigning both FX units to a deck requires duplicating Super/Mix knobs and FX Pfl button. In that case the whole left/right concept of Shade becomes inconsistent and there's a lot of potential for confusion.

Long story short: I won't implement it in this PR.

**Comment by Be-ing** [comment]
Right, this is why the effect unit controls should not be in the deck area. They are not deck controls.

**Comment by ronso0** [comment]
Yes, but the position of those controls matches the Shade concept as I understand it: they are shortcuts so you can keep the advanced control interface collapsed/hidden and have a usable library on small screens. IMO this makes sense here.

**Comment by ronso0** [comment]
I'm done here, please test.

**Comment by Be-ing** [comment]
At minimum width, the rightmost + icon for effect focusing overlaps the border of the effect unit slightly:
image

**Comment by Be-ing** [comment]
There's another issue with borders at minimum width for the loop section:
image

**Comment by ronso0** [comment]
> At minimum width, the rightmost + icon for effect focusing overlaps the border of the effect unit slightly:

> There is no black outer border for the loop sections at minimum width:

..better than it was before, when the Mixxx window was at minimal size on the most uncomfortable OS and theme config mentioned in the skin wiki (WindowsXP, 1024px, window maximized but not fullscreen).
Fixed. There are a few other issues but for now I can't continue to fix Shade for that configuration, it's ready for 2.1.

shade-fixed-minimum-width

It would be nice if anyone could check all the tooltips and let me know about issues.

**Comment by daschuer** [comment]
Great, thank you very much for your help. LGTM 
The only pending issue is the routing buttons. 
I think about putting the deck routing button next to the bus routing button. But it is not obvious when should one use which.
